lightningd/peer_control.c

```diff
@@ -1627,6 +1632,51 @@ void peer_last_tx(struct peer *peer, struct bitcoin_tx *tx,
 	peer->last_tx = tal_steal(peer, tx);
 }
 
+static void json_sign_last_tx(struct command *cmd,
```

Comment:
Ouch, this is really dangerous. Any chance we can hide this in releases behind a couple of compile guards? See #176

Reply:
Yes, definitely!  Will add FIXME here.